simple-no-auth/src/test/java/com/fga/example/OpenFgaContainerConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package com.fga.example;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.time.Duration;
+
+@TestConfiguration(proxyBeanMethods = false)
+class OpenFgaContainerConfiguration {
+
+	@Bean
+	GenericContainer<?> openFgaContainer(DynamicPropertyRegistry registry) {
+		var result = new GenericContainer<>("openfga/openfga:latest")
+				.withCommand("run")
+				.waitingFor(Wait.forHttp("/playground").forPort(3000).withStartupTimeout(Duration.ofMinutes(2)))
+				.withEnv("OPENFGA_HTTP_ADDR","0.0.0.0:4000")
+				.withExposedPorts(4000, 8081, 3000);
+		registry.add("openfga.fgaApiUrl", () -> {
+			Integer httpPort = result.getMappedPort(4000);
+			return "http://localhost:"+httpPort;
```

Comment:
Consider using `result.getHost()` instead of a hardcoded `localhost` to support even more environments 👍

Reply:
Thanks for the feedback! I've resolved this https://github.com/jimmyjames/fga-spring-examples/pull/4/commits/f810277bf0e45f25cb002c19ecae51dd64000ba3